src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -19,7 +19,11 @@
     <None Remove="UI\Images\NodeStates\hidden-64px.png" />
     <None Remove="UI\Images\NodeStates\info-64px.png" />
     <None Remove="UI\Images\warning_16px.png" />
+    <None Remove="Views\GuidedTour\HtmlPages\nodePackage.html" />
     <None Remove="Views\Menu\TrustedPathView.xaml" />
+    <None Remove="Views\GuidedTour\HtmlPages\getStartedOnboarding.html" />
```

Comment:
Can you double check all these resources, I think I removed some of them, see: https://github.com/DynamoDS/Dynamo/pull/12852/files

Reply:
@QilongTang I've removed the non existing html resource file, see the next commit:
https://github.com/DynamoDS/Dynamo/pull/12860/commits/390e80aaa5ddfcddc36898541d99f4063a5a49fa